junifer/datagrabber/datalad_base.py

```diff
@@ -46,17 +46,17 @@ class DataladDataGrabber(BaseDataGrabber):
         Removes the datalad dataset from the `datadir`. This method is called
         automatically when the datagrabber is used within a context.
 
+    See Also
+    --------
+    BaseDataGrabber
+        For the abstract base class of any datagrabber.
+
     Notes
     -----
     By itself, this class is still abstract as the `__getitem__` method relies
     on the parent class `BaseDataGrabber.__getitem__` which is not yet
     implemented. This class is intended to be used as a superclass of a class
     with multiple inheritance.
-
```

Comment:
I remember this section needed to be at last according to numpydoc, or am I mistaken?